packages/ui/src/components/Command/GenerateSQL/GenerateSQL.tsx

```diff
@@ -400,6 +269,23 @@ Postgres SQL query:
         <div className="[overflow-anchor:auto] h-px w-full"></div>
       </div>
       <div className="absolute bottom-0 w-full bg-scale-200 py-3">
+        {allowSendingSchemaMetadata && (
+          <div className="flex items-center justify-between px-6 py-3">
+            <div>
+              <p className="text-sm">
+                Include table names, column names and their corresponding data types in prompt
+              </p>
+              <p className="text-sm text-scale-1100">
+                This will generate answers that are more relevant to your project's schema
+              </p>
+            </div>
+            <Toggle
+              disabled={!isOptedInToAI}
+              checked={includeSchemaMetadata}
+              onChange={() => setIncludeSchemaMetadata(!includeSchemaMetadata)}
```

Comment:
to avoid stale closure errors, this should be `onChange={() => setIncludeSchemaMetadata(prev => !prev)}`